initial add of working lambda preProcessor-home

mabbason opened the pull request:
Co-authored-by: Katarina Rosiak <katarinarosiak@gmail.com>
Co-authored-by: Tim Dronkers <tdronkers@gmail.com>
Co-authored-by: Scott Graham <scttgrhm7+public@gmail.com>

# preProcessor-home
I zipped the lambda code in this PR and uploaded it, then confirmed that it is working. See screenshot below for confirmation of the function run in the infrastructure.

![2022-07-13_15h51_03](https://user-images.githubusercontent.com/32547845/178820947-f156a677-1f4d-4d22-97f2-3039b45a5547.png)

scogra17 commented on preProcessor-home/.eslintrc.json:
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "airbnb"
```
For consistency, we've been using (it shouldn't change any of the rules, it's just more explicit): 
```
{
  "env": {
    "browser": false,
    "commonjs": true,
    "es2021": true
  },
  "extends": [
    "airbnb-base"
  ],
  "parserOptions": {
    "ecmaVersion": "latest"
  },
  "rules": {}
}
```

mabbason replied:
I updated in most recent commit

mabbason commented:
I updated the eslint file and cleaned a couple console.logs out of the function. See attached screenshot below for latest successful logs messages regarding most recent changes
![2022-07-13_18h43_09](https://user-images.githubusercontent.com/32547845/178849701-991e6ecf-0cbe-43bd-9d8d-12852a5260fc.png)
.
